api/schemas.py

```diff
@@ -12,7 +12,7 @@ class FieldSchema(Schema):
     @pre_dump
     def serialize_input_options(self, data, many):
         """Convert input_options from bytes obj to dict."""
-        if data.input_options:
+        if data.input_options and type(data.input_options) != dict:
             input_options = data.input_options.decode().replace("'", '"')
```

Comment:
Wrapping this in a try/except block that catches decoding errors would be a good idea.

Reply:
Addressed in [74beac1](https://github.com/wikimedia/toolhunt/pull/55/commits/74beac181f92ee865959d0862247e0f9e96fa202)